pyramus/src/main/java/fi/otavanopisto/pyramus/util/dataimport/scripting/api/CourseAPI.java

```diff
@@ -50,6 +53,12 @@ public Long create(Long moduleId, Long typeId, String name, String nameExtension
     if (subject == null) {
       throw new InvalidScriptException("Subject by code '" + subjectCode + "' not found.");
     }
+    
+    // TODO Default organization?
```

Comment:
One day the import view should ask for the organization and pass it to the api classes.